projects/step-core/src/lib/modules/basics/shared/generate-api-key-strategy.ts

```diff
@@ -1,3 +1,7 @@
+import { Observable } from 'rxjs';
+
 export interface GenerateApiKeyStrategy {
-  showGenerateApiKeyDialog(): void;
+  showGenerateApiKeyDialog(): Observable<any>;
+  getServiceAccountTokens(): Observable<Array<any>>;
```

Comment:
The template below shows, that we know something about token's structure.
At least it should contain: id, label, creationTime and expirtyTime.
It would be nice to define such interface and use it in OS.
The method notation it this case could be written as generic:
```
getServiceAccountTokens<T extends BaseAccountToken>(): Observable<T[]>;
```